app/permissions/Permissions.scala

```diff
@@ -1,40 +1,32 @@
 package permissions
 
-import com.gu.editorial.permissions.client._
+import com.amazonaws.auth.{AWSCredentialsProvider, DefaultAWSCredentialsProviderChain}
+import com.gu.permissions.{PermissionDefinition, PermissionsConfig, PermissionsProvider}
 import services.Config
 
-import scala.concurrent.Future
+object Permissions {
+  val app = "tag-manager"
 
-object Permissions extends PermissionsProvider {
+  val TagEdit: PermissionDefinition = PermissionDefinition("tag_edit", app)
+  val TagAdmin: PermissionDefinition = PermissionDefinition("tag_admin", app)
+  val CommercialTags: PermissionDefinition = PermissionDefinition("commercial_tags", app)
+  val TagUnaccessible: PermissionDefinition = PermissionDefinition("tag_no_one", app)
 
-  lazy val TagEdit = Permission("tag_edit", "tag-manager", PermissionDenied)
-  lazy val TagAdmin = Permission("tag_admin", "tag-manager", PermissionDenied)
-  lazy val CommercialTags = Permission("commercial_tags", "tag-manager", PermissionDenied)
-  lazy val TagUnaccessible = Permission("tag_no_one", "tag-manager", PermissionDenied)
-
-  lazy val all = Seq(TagAdmin)
-
-  implicit def config = PermissionsConfig(
-    app = "tag-manager",
-    all = all,
-    s3BucketPrefix = Config().permissionsStage,
-    s3Region = Some("eu-west-1")
+  private val permissionDefinitions = Map(
+    "tag_edit" -> TagEdit,
+    "tag_admin" -> TagAdmin,
+    "commercial_tags" -> CommercialTags,
+    "tag_no_one" -> TagUnaccessible
   )
 
-  def testUser(permission: Permission)(email: String): Future[PermissionAuthorisation] = {
-    println("Permissions for: " + email)
-    implicit val permissionsUser: PermissionsUser = PermissionsUser(email)
-
-    Permissions.get(permission)
-  }
+  private val credentials: AWSCredentialsProvider = new DefaultAWSCredentialsProviderChain()
 
-  def getPermissionsForUser(email: String): Future[Map[String, Boolean]] = {
-    implicit val permissionsUser: PermissionsUser = PermissionsUser(email)
+  private val permissions: PermissionsProvider = PermissionsProvider(PermissionsConfig(Config().permissionsStage, Config().aws.region, credentials))
 
-    Permissions.list.map(_.filter(_._1.app == "tag-manager").flatMap( _ match {
-      case (p: Permission, PermissionGranted) => Map(p.name -> true)
-      case (p: Permission, PermissionDenied) => Map(p.name -> false)
-    }))
+  def testUser(email: String)(permission: PermissionDefinition): Boolean = {
```

Comment:
Non-blocking: is there a reason we've swapped the curried arguments here? It might be quite useful to keep permissions as the first argument, as we can create instances of the function that are bound to particular permissions. (In the examples below I've removed the `Future` requirement per my other comment.)

For example, implementing `testAction` in `PermissionsActionCheck` is simpler:

```scala
val testAccess: String => Future[Boolean] =
    email => Future.successful(Permissions.testUser(email)(permission))
```

becomes

```scala
val testAccess: String => Boolean = Permissions.testUser(permission)
```

It might also help elsewhere – for example, in `SectionSpecificPermissions`, we have the line 

```scala
def commonTestAccess: PermissionDefinition => String => Future[Boolean] =
    permission => email => Future.successful(Permissions.testUser(email)(permission))
```

which could become

```scala
def commonTestAccess: PermissionDefinition => String => Boolean = Permissions.testUser
```

Reply:
This was unintentional, thanks for spotting @jonathonherbert!